test/lib/org.test.ts

```diff
@@ -83,3 +83,31 @@ describe('listProjects', () => {
     });
   }, 5000);
 });
+
+
+describe('listTargets', () => {
+  const OLD_ENV = process.env;
+  process.env.SNYK_API = SNYK_API_TEST;
+  process.env.SNYK_TOKEN = process.env.SNYK_TOKEN_TEST;
+  const requestManager = new requestsManager({
+    userAgentPrefix: 'snyk-api-import:tests',
+  });
+  afterAll(async () => {
+    process.env = { ...OLD_ENV };
+  }, 1000);
+  it('list the targets in a given Org', async () => {
```

Comment:
please add a test for an error, for example 1 page returns 500

Reply:
yeah mocks allow for good control of this, we have the system test that will for real anyway so we can have a mix